### DISCUSSION TITLE
checkbox in kanban - how to?

### DISCUSSION THREAD

**Discussion by dazilla3** [mgmeyers/obsidian-kanban#572] [Ideas & Feature Requests]
sorry maybe i'm blind, i can't find the solution and maybe someone can help me.
How can i add a checkbox in a Kanban Task like in the screenshot? Im not able to.

![Bildschirmfoto 2022-06-30 um 08 28 22](https://user-images.githubusercontent.com/48415304/176632450-2cb3ea8f-0eee-443d-b7cb-408dc6730b06.png)

thx for your help!

**Comment by vereym** [comment]
Just checked and you can achieve this by typing the normal markdown syntax for it.
So `- [ ] my task` enables you to do this. 🙂

**Reply by hardwarepirate** [reply]
> Just checken and you can achive this by typing the normal markdown syntax for it. So `- [ ] my task` enables you to do this. 🙂

for me neither 
`[ ]
- [ ]
    -
    [ ]`
is working

how do you do it ?
with either checkboxes disabled or enabled in kanban settings, it solely either shows checkboxes for all list entries or for none, despite my md edits.
also it would be superb to combine the nested checkboxes with nested lists, like requested here:
https://github.com/mgmeyers/obsidian-kanban/discussions/99#discussioncomment-779821
@yoursamlan @mgmeyers @dazilla3

**Reply by vereym** [reply]
Hey @hardwarepirate I'm afraid I don't quite understand your issue.
For me it works both ways.

When I enable "Display card checkbox" with this markdown: `- [ ] some task` (note the \` character is only there to add code styling in github so in obsidian you would write just - [ ] some task)
I get this 
![image](https://user-images.githubusercontent.com/74200990/232538439-3bb86bc0-399c-42da-8e5e-26d1073a6372.png)

And when I disable the checkoxes I get this:

![image](https://user-images.githubusercontent.com/74200990/232538693-5de240d2-2940-45ab-aebd-48d136e8eb84.png)

Hope that solves your issue :)

**Reply by mbbroberg** [reply]
I hold Shift, press Enter, then on a new line add a `- [ ]` at the start of the line. That will get a single card with multiple checkboxes. Good luck!

**Comment by SkinnyDennis** [comment]
Checkboxes are not displaying for me on Kanban pages either. They do work on regular Obsidian pages. I am using Obsidian version 1.1.16 and Kanban version 1.5.2 for Windows.

Thank you for such a great tool. I am a new user exploring moving from Onenote to Obsidian, and the Kanban plugin is a definite plus.

![image](https://user-images.githubusercontent.com/105157973/234906390-fc0fa083-41ce-42bc-96cd-7faa17df5bf8.png)

**Comment by SkinnyDennis** [comment]
I found the solution to my checkbox problem, which may not be a bug but just my unfamiliarity with Obsidian in general.

The checkboxes do not show when they are part of sub list to an uncheckboxed item, but if in the above example there is a checkbox in front of the top-level item ("test card", "enhanced search filtering") then they display properly.
